Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -528,19 +528,35 @@ class DropDownMenu
 
     popUpDiv.style.top = top + 'px';
```

Comment:
Fix for context menu behavior, right now, it does not reposition to the left of the mouse, but rather goes into overflow.

For context menu, it may be best that the menu is stuck to the right border first and then grows past the mouse cursor towards the left border (= inverse behavior of dropdown menu popup). Reasoning: By default context menu opens to the right of the mouse cursor, dropdown menu popup opens to the left of the drop down button.